nettacker/core/arg_parser.py

```
@@ -612,9 +612,8 @@ def parse_arguments(self):
             options.targets = list(set(options.targets.split(",")))
         if options.targets_list:
             try:
-                options.targets = list(
-                    set(open(options.targets_list, "rb").read().decode().split())
-                )
+                with open(options.targets_list, "rb") as f:
+                    options.targets = list(set(f.read().decode().split()))
             except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)